ws/templates/help/leaders/ws_gear.html

```diff
@@ -29,9 +29,9 @@ <h4>Above-treeline Gear</h4>
 <div class="well" id="traction">
   <h4>Traction/Floatation</h4>
   <ul>
-    <li>Crampons</li>
     <li>Microspikes</li>
     <li>Snowshoes</li>
+    <li>Crampons</li>
```

Comment:
Any reason to not just keep these in alphabetical order?

Reply:
oh - lol totally didn't notice that was the order. just thought microspikes are the primary traction so was putting it first. no strong opinion

Reply:
🤷 whichever you prefer. Makes sense to include these in order of likelihood to be actually used (though probably nice to specify that plainly if we do order that way!)